### DISCUSSION TITLE
can I access resources deeply nested by CRUD routes

### DISCUSSION THREAD

**Discussion by Firkraag** [mockoon/mockoon#1176] [Help] [answered]
For example, I have a data:
```json
[
  {
    "title": "Section 1",
    "id": 1,
    "todos": [
      {
        "personID": 1,
        "personName": "Person 1",
        "completed": false,
        "title": "abdsafld",
        "id": 1
      }
    ]
  }
]
```
and I have CRUD route /sections, can I access todos by URL /sections/1/todos?

**Comment by 255kb** [comment, marked as answer]
No, for now you can only access first level resources, by using their ids, not sub-resources. You can see [here](https://mockoon.com/docs/latest/api-endpoints/crud-routes/#list-of-routes-and-operations) the list of routes that are automatically created. 

Feel free to open an issue and contribute as this would be a great feature.
